Modules/Nonunit/Review/test/itkImageFunctionTest.cxx

```diff
@@ -134,8 +134,8 @@ itkImageFunctionTest(int, char *[])
   image->SetRegions(region);
   image->Allocate();
 
-  auto origin = MakeFilled<ImageType::PointType>(0.0);
```

Comment:
For zero-filling, I would suggest using `{}` instead:

    ImageType::PointType origin{};